statsmodels/sandbox/nonparametric/kernel_extras.py

```diff
@@ -75,7 +75,7 @@ class TestFForm(object):
     References
     ----------
     See Racine, J.: "Consistent Significance Testing for Nonparametric
-    Regression" Journal of Business \& Economics Statistics.
+    Regression" Journal of Business \\& Economics Statistics.
```

Comment:
Why does this need any escaping?

Reply:
I'd be happy without; assumed previous author had a reason.  Will change.